dask/array/routines.py

```diff
@@ -41,8 +41,8 @@
 
 
 @derived_from(np)
-def array(x, dtype=None, ndmin=None):
-    x = asarray(x)
+def array(x, dtype=None, ndmin=None, *, like=None):
```

Comment:
Can you remind me what the `*` does here ?

Reply:
It makes kwargs to its right a keyword-only arguments, as documented in [PEP-3102](https://www.python.org/dev/peps/pep-3102/).

Reply:
To clarify it further, the `like` is a keyword only argument. From [NEP-35](https://numpy.org/neps/nep-0035-array-creation-dispatch-with-array-function.html#motivation-and-scope):

> Support for Python 2.7 has been dropped since NumPy 1.17, therefore we make use of the keyword-only argument standard described in PEP-3102 to implement `like=`, thus preventing it from being passed by position.

Reply:
Thanks